### ISSUE TITLE
Do not drop Runed Tablet from Thaumcraft

### ISSUE THREAD

**Issue by Kostia-2001** [GTNewHorizons/NewHorizons#2487]
#### Which modpack version are you using?

#
#### If in multiplayer; On which server does this happen? 

#
#### What did you try to do, and what did you expect to happen?

#
#### What happend instead? (Attach screenshots if needed)

#

**Comment by Dream-Master** [comment]
Its a Thaumcraft item or a addon mod item ?  @Kostia-2001 need more infos please and pics. Thanks

**Comment by Kostia-2001** [comment]
I read on the Internet that this is due to a plugin that cleans item faster.

**Comment by Kostia-2001** [comment]
No, no one could take him. I immediately opened it and entered it.

**Comment by Dream-Master** [comment]
Can be happend so ask the admin to change it. Or whitelist items that not get wiped.

**Comment by Kostia-2001** [comment]
The quest says that you need to find it, but I did not find it on the pedestal.

**Comment by Dream-Master** [comment]
1. Let the dim wipe from a admin adn try it
2. Change the time items get wiped on the server and make a whitelist which item exclude from it.
3. Try in sp in creative and see if it is a bug
4. report here back what you find out.

**Comment by Kostia-2001** [comment]
Ok.

**Comment by Kostia-2001** [comment]
This Key added in shop.

**Comment by Dream-Master** [comment]
Oh ok then we can close this issue i guess ?

**Comment by Dream-Master** [comment]
@Kostia-2001

**Comment by Dream-Master** [comment]
reopen it if you have additional infos.
